packages/x-date-pickers/src/models/index.ts

```diff
@@ -6,5 +6,7 @@ export * from './adapters';
 export * from './common';
 export * from './pickers';
 
+export type { PickerLayoutOwnerState } from '../PickersLayout/PickersLayout.types';
```

Comment:
Exporting from here to allow importing in `@mui/x-date-pickers-pro`.
Or would you suggest moving it beside the `PickerOwnerState`?

Reply:
Yeah, thanks for the tip. 👍 
Was exploring the exporting from `/models`, but that also causes circular dependencies without major changes to usage. 🙈